composer.json

```diff
@@ -22,7 +22,7 @@
         "pestphp/pest-plugin": "^0.3",
         "pestphp/pest-plugin-coverage": "^0.3",
         "pestphp/pest-plugin-init": "^0.3",
-        "phpunit/phpunit": ">= 9.3.7 <= 9.4.0"
+        "phpunit/phpunit": ">= 9.3.7"
```

Comment:
```suggestion
Can you update it to use an upper limit so this can be merged? We are extending internal classes so can't just allow any version.
                "phpunit/phpunit": ">= 9.3.7 <= 9.4.1"
```